src/main/java/com/testsigma/sdk/EnvironmentData.java

```diff
@@ -2,16 +2,21 @@
 
 import lombok.Data;
 import lombok.RequiredArgsConstructor;
-import org.json.JSONObject;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
 
 @Data
 @RequiredArgsConstructor
 public class EnvironmentData {
 
-    private final JSONObject parameters;
+    private final List<VariableData> variables;
```

Comment:
@udayareddyvempalli As we discussed, this is a breaking change. I feel very less customers use this. But can we think of something to inform support team to see if these kind of addons are present anywhere. Atleast major customers like NTUC, hansard.

Reply:
Sure Kasyap